rules/Privatization/Reflection/ClassConstantsResolver.php

```diff
@@ -34,8 +35,7 @@ public function getClassConstantNamesToValues(string $classWithConstants): array
         $classReflection = $this->reflectionProvider->getClass($classWithConstants);
         $nativeReflection = $classReflection->getNativeReflection();
 
-        $constantNamesToValues = $nativeReflection->getConstants();
-        $this->cachedConstantNamesToValues = $constantNamesToValues;
+        $this->cachedConstantNamesToValues = $nativeReflection->getConstants();
```

Comment:
`$constantNamesToValues` is used after assigned, so `ExprUsedInNextNodeAnalyzer->isUsed($expr)` check is needed.

Reply:
I'll fix the CI first :). I'll ask you for review after. Not neccessary to comment now, the CI is showing the bugs.